google/datalab/bigquery/_query.py

```diff
@@ -154,11 +154,11 @@ def sql(self):
     return self._expanded_sql()
 
   @property
-  def scripts(self):
+  def udfs(self):
     """ Get the code for any Javascript UDFs used in the query. """
     return self._code
```

Comment:
this should return `self._udfs`, i forgot to make that change. we should remove `self._code` now since we don't support legacy SQL anymore.